cellprofiler/utilities/cpjvm.py

```diff
@@ -67,7 +67,7 @@ def get_patcher_args(class_path):
 
     patchers = filter(
         (lambda x:os.path.split(x)[1].startswith("prokaryote")), class_path)
-    if len(patchers) > 0 and False:
```

Comment:
What did `and False` check? It would’ve failed regardless.

Was this intended behavior to circumvent some other issue?